iset.mm

```diff
@@ -139302,10 +139302,19 @@ is a function if the structure (without the empty set) is a function.
   baseid $p |- Base = Slot ( Base ` ndx ) $=
     ( cbs c1 df-base 1nn ndxid ) ABCDE $.
 
-  $( Index value of the base set extractor.  (Normally it is preferred to work
-     with ` ( Base `` ndx ) ` rather than the hard-coded ` 1 ` in order to make
-     structure theorems portable.  This is an example of how to obtain it when
-     needed.)  (New usage is discouraged.)  (Contributed by Mario Carneiro,
+  $( Index value of the base set extractor.
+
+     Normally it is preferred to work with ` ( Base `` ndx ) ` rather than the
+     hard-coded ` 1 ` .  For example, using theorems such as ~ baseid and
+     ~ basendxnn .
```

Comment:
"Normally ... preferred" sounds too coy, and the second "sentence" is not a sentence. Maybe:
```
Use of this theorem is discouraged since the particular value 1 for the index is an
implementation detail.  It is generally sufficient to work with ` ( Base `` ndx ) ` and
use theorems such as ~ baseid and ~ basendxnn .
```
?

Reply:
Some of that is text that was present before, so I don't know if anyone is attached to it, but the suggestion here seems fine with me.